src/AccessibilityInsights.CommonUxComponents/Controls/CustomControlOverridingAutomationPeer.cs

```diff
@@ -26,43 +28,53 @@ public class CustomControlOverridingAutomationPeer: UserControlAutomationPeer
         /// </summary>
         private string LocalizedControlType;
 
+        private AutomationControlType ControlType;
+
+        /// <summary>
+        /// Whether to hide all children automation peers
```

Comment:
This comment adds no information over the name of the property; why include it? (same for the other existing doc comments on the private fields above it)